src/playwrightTestCLI.ts

```diff
@@ -128,6 +129,60 @@ export class PlaywrightTestCLI {
     await reporterServer.wireTestListener(reporter, token);
   }
 
+  async debugTests(locations: string[], testDirs: string[], options: PlaywrightTestRunOptions, reporter: reporterTypes.ReporterV2, token: vscodeTypes.CancellationToken): Promise<void> {
+    const configFolder = path.dirname(this._config.configFile);
+    const configFile = path.basename(this._config.configFile);
+    const escapedLocations = locations.map(escapeRegex);
+    const args = ['test',
+      '-c', configFile,
+      ...escapedLocations,
+      '--headed',
+      ...(options.projects || []).map(p => `--project=${p}`),
+      '--repeat-each', '1',
+      '--retries', '0',
+      '--timeout', '0',
+      '--workers', '1'
```

Comment:
Shouldn't we read these options from `options` instead of hardcoding them?

Reply:
Only workers is a part of PlaywrightTestRunOptions, the rest are hard-coded.